features/admin/moderator_responds_to_petition.feature

```diff
@@ -99,7 +99,6 @@ Feature: Moderator respond to petition
     When I look at the next petition on my list
     And I reject the petition with a reason code "Not Parliament/Government’s responsibility"
     Then the petition is not available for signing
-    But the petition is still available for searching or viewing
```

Comment:
Probably need to delete the previous step and add a new one to test it's not visible